backend/app/api/v1/endpoints/auth.py

```
@@ -0,0 +1,66 @@
+    LoginRequest,
+    PasswordUpdateRequest,
+)
+from app.utils.response_handler import ResponseHandler
+from app.middlewares.auth_middleware import auth_required
+router = APIRouter()
+
+
+@router.post("/signup")
+async def signup(body: UserCreate):
+    """Create a new user account."""
+    user_response = AuthService().signup(body)
```

Ruff (0.14.10)

Local variable `user_response` is assigned to but never used

Remove assignment to unused variable `user_response`

(F841)